src/global_mapper/global_mapper_params.h

```diff
@@ -8,16 +8,32 @@ class Params {
  public:
   Params();
 
-  std::vector<double> voxel_xyz_min_;
-  std::vector<double> voxel_xyz_max_;
-  std::vector<double> voxel_resolution_;
+  // pixel map
+  double pixel_xy_min_[2];
+  double pixel_xy_max_[2];
```

Comment:
That *is* what the GSG says, but by the GSG this should be a struct since it's POD , in which case the members should not have trailing underscores: 
https://google.github.io/styleguide/cppguide.html#Structs_vs._Classes

The trailing underscores are mainly to signify that a member is private and that you probably shouldn't be accessing it directly (this is how Python implements private members).